automation/taskcluster/docker/build.dockerfile

```diff
@@ -0,0 +1,126 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM ubuntu:bionic-20181018
+
+MAINTAINER Nick Alexander "nalexander@mozilla.com"
+
+#----------------------------------------------------------------------------------------------------------------------
```

Comment:
Nit: Any reason to keep the long `-----`?

Reply:
I've dropped these throughout.  A vestige from Android Component's base.